src/Swift.Bindings/src/Emitter/StringCSharpEmitter/Handler/MethodHandler.cs

```diff
@@ -278,7 +293,14 @@ public Signature Build()
         /// <param name="returnType">The return type.</param>
         private void SetReturnType(string returnType)
         {
-            _returnType = returnType;
+            if (returnType == "()")
```

Comment:
I think we should document void handling. This can be done separately.

Reply:
There was a bug in type registration, this special handling is not needed.